### ISSUE TITLE
Possible data incompleteness of source information in Person.csv and Institution.csv

### ISSUE THREAD

**Issue by whalekeykeeper** [readchina/ReadAct#460]
There are `source` or `source_1` or `source_2` columns in `Person.csv` and `Institution.csv`.
- It seems that they should be filled in with more information.
- Plus, it is also negotiable that which language should we use to search on Wikipedia or other sites.

For example, in `Person.csv`, we can see the following from those two files:
|person_id|family_name|first_name|name_lang|sex|rustication|birthyear|deathyear|alt_name|place_of_birth|social_position|neibu_access|source_1|page_1|source_2|page_2|note|created|created_by|last_modified|last_modified_by|
|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|
|AG0004|芒|克|zh|male|1|1950|XXXX|姜世伟|SP0023|||W0423|75f 79f|W0442|183ff||2017-07-05|LH|2020-04-02|DP|

And what is expected might be:
|person_id|family_name|first_name|name_lang|sex|rustication|birthyear|deathyear|alt_name|place_of_birth|social_position|neibu_access|source_1|page_1|source_2|page_2|note|created|created_by|last_modified|last_modified_by|
|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|
|AG0004|芒|克|zh|male|1|1950|XXXX|姜世伟|SP0023|||https://zh.wikipedia.org/wiki/%E8%8A%92%E5%85%8B|75f 79f|W0442|183ff||2017-07-05|LH|2020-04-02|DP|

Similarly, for the following institution in`Institution.csv`:
|inst_id|inst_name|source|
|-|-|-|
|AG0636 | 八一电影制片厂||

It might be expected to be:
|inst_id|inst_name|source|
|-|-|-|
|AG0636 | 八一电影制片厂|https://zh.wikipedia.org/wiki/%E4%B8%AD%E5%9B%BD%E4%BA%BA%E6%B0%91%E8%A7%A3%E6%94%BE%E5%86%9B%E6%96%87%E5%8C%96%E8%89%BA%E6%9C%AF%E4%B8%AD%E5%BF%83%E7%94%B5%E5%BD%B1%E7%94%B5%E8%A7%86%E5%88%B6%E4%BD%9C%E9%83%A8|

**Comment by duncdrum** [comment]
not sure if this is still an issue can you elaborate what a good fix would look like

**Comment by whalekeykeeper** [comment]
> not sure if this is still an issue can you elaborate what a good fix would look like
1. There is no need to fix the old source/source-related columns. We have made use of the existed wikipedia links already when we ran the entire `Person.csv` with scripts in ReadActor and added all the qids at once into `Agent.csv`.
2. If we do not introduce more wikipedia link in source/source-related columns for future data, we don't need to modify any code. And if any user find an entity on Wikidata,  I would expect the user to add the qid into Agent table than to add wikipedia link in Person/Institution table.
